src/app.html

```diff
@@ -11,8 +11,28 @@
     />
     <title>Good First Issue Finder</title>
     %svelte.head%
+    <style>
+      body {
+        padding: 25px;
+        background-color: white;
+        color: black;
+        font-size: 25px;
+      }
+      
+      .dark-mode {
+        background-color: black;
+        color: white;
+      }
+      </style>
   </head>
   <body>
+    <script>
+    function myFunction() {
```

Comment:
Could you please also make this function name a bit more descriptive? `myFunction` doesn't say anything about what the function does.

Reply:
Thanks @Cahllagerfeld for your suggestions 
I will make the required changes